src/overlays/actors/ovl_En_Dodongo/z_en_dodongo.c

```diff
@@ -9,7 +17,34 @@ void EnDodongo_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnDodongo_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnDodongo_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-#if 0
+void func_80877424(EnDodongo* this, GlobalContext* globalCtx);
+void func_80877500(EnDodongo* this, GlobalContext* globalCtx);
+void func_8087784C(EnDodongo* this, GlobalContext* globalCtx);
+void func_80877D90(EnDodongo* this, GlobalContext* globalCtx);
+void func_808786C8(EnDodongo* this, GlobalContext* globalCtx);
+void func_80877E60(EnDodongo* this, GlobalContext* globalCtx);
+void func_808787B0(EnDodongo* this, GlobalContext* globalCtx);
+void func_808785B0(EnDodongo* this, GlobalContext* globalCtx);
+void func_80878424(EnDodongo* this, GlobalContext* globalCtx);
+
+void func_808773C4(EnDodongo* this);
+void func_80877494(EnDodongo* this);
+void func_808777A8(EnDodongo* this);
+void func_80877D50(EnDodongo* this);
+void func_80878354(EnDodongo* this);
+void func_80878724(EnDodongo* this);
+void func_8087864C(EnDodongo* this);
```

Comment:
Doesn't really matter, but I'd prefer these ordered by address rather than separated like this, given that they're setup funcs for the actionfuncs